.cirrus.yml

```diff
@@ -0,0 +1,64 @@
+---
+# We use Cirrus for Vagrant tests, because macOS instances of GHA
+# are too slow and flaky, and Linux instances of GHA do not support KVM.
+
+compute_engine_instance:
+  image_project: cirrus-images
+  image: family/docker-kvm
+  platform: linux
+  nested_virtualization: true
+  # CPU limit: `16 / NTASK`: see https://cirrus-ci.org/faq/#are-there-any-limits
+  cpu: 8
+  # Memory limit: `4GB * NCPU`
+  memory: 32G
+
+task:
+  timeout_in: 30m
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    HOME: /root
+    # yamllint disable rule:key-duplicates
+    matrix:
+      DISTRO: fedora34
+      DISTRO: centos7
+  info_script: |
+    uname -a
+    echo "-----"
+    cat /etc/os-release
+    echo "-----"
+    cat /proc/cpuinfo
+    echo "-----"
+    df -T
```

Comment:
I'd like to also see systemd version (either `systemd --version` or `rpm -q systemd`), as it's currently done in GHA.

Reply:
nevermind, I see it down there

Reply:
Renamed to `host_info_script` and `guest_info_script`